lhotse/recipes/mobvoihotwords.py

```diff
@@ -0,0 +1,145 @@
+import json
+import logging
+import shutil
```

Comment:
I'm just wondering whether it would be obvious to the user how to use this.
I guess there is the same issue in the other recipes... dunno whether this could be addressed with a README, or a pointer at the top of each file, or something?  I'm guessing there must be a standard pattern.

Reply:
In the other files, we inserted a module-level docstring on top as a description. Yiming, could you add it? 

In the wider context of Lhotse, it's not ideal - there are at least two functions that should have the same description in their docstrings (`prepare_X` and the CLI `lhotse prepare X`). I'll add an issue, I think we can do it with decorators that attach docstrings dynamically (similarly like the `transformers` library does [here](https://github.com/huggingface/transformers/blob/master/src/transformers/modeling_longformer.py#L1071))